File: 1sem/Python/Lista2/Colevati/Ex04.py
# pegar 30 reais // média // qtd acima da média // posições dos vals abaixo da média

def main():
    nums: float = [0.0]*30
    med: float = 0.0
    acima: int = 0
    abaixo: int = []
    j: int = 0

    for i in range(len(nums)):
        nums[i] = float(input('Digite o valor: '))
        med += nums[i]

    med = med/len(nums)
    i=0

    for i in range(len(nums)):
        if (nums[i] > med):
            acima+=1
        if (nums[i] < med):
            abaixo.append(i)

    print('A média é {:.2f}'.format(med))
    print('Quantidade de números acima da média:',acima)
    print('Posições dos números abaixo da média: {}'.format(abaixo))
    print()

if __name__ == '__main__':
    main()

File: 2sem/Java/Lista 2/prj_java10-09-2026/src/Ex04.java
import javax.swing.JOptionPane;
public class Ex04 {
    public static void main (String args[]){
        float v[] = new float[30];
        float med = 0;
        int qa = 0; // Notas acima da média
        String pos = ""; // Posições abaixo
        
        for (int i=0;i<30;i++){
            v[i] = Float.parseFloat(JOptionPane.showInputDialog("Digite o valor real do "+(i+1)+" termo: "));
            med += v[i];
        }
        
        med = med/30;
        
        for (int i=0;i<30;i++){
            if (v[i] > med){
                qa++;
            }
            if (v[i] < med){
                pos += i + ", ";
            }
        }
        
        System.out.println("Média: "+med);
        System.out.println("Qtd. valores acima da média: " +qa);
        System.out.println("Posições de valores abaixo da média: "+pos);
        System.out.println("Valores informados: ");
        for (int i=0;i<30;i++){
            if (i%9==0){System.out.println("");}
            System.out.print(v[i]+" ");
        }
    }
}
